Article/Article.js

```diff
@@ -3,27 +3,31 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.domElement = domElement;
+    // create a reference to the ".expandButton" class.
+    this.expandButton = domElement.querySelector(".expandButton");
+    this.expandButton.innerHTML = "expand";
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.addEventListener("click", this.expandArticle);
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
   }
 
   expandArticle() {
```

Comment:
The extra `open` class seems redundant if you can apply it all to a `article--open` class.. and we use the double dash between a component name and an ACTION when following the BEM naming convention for CSS: https://css-tricks.com/bem-101/

You can also simplify the conditional with a ternary operator:

```
  expandArticle() {
    // Using our reference to the domElement, toggle a class to expand or hide the article.
    this.domElement.classList.toggle("article--open");
    
    this.expandButton.textContent === "Expand"
      ? (this.expandButton.textContent = "Close")
      : (this.expandButton.textContent = "Expand");
  }
```